common/v2/services/Store/BalanceService.tsx

```diff
@@ -7,12 +7,13 @@ import {
 } from '@mycrypto/eth-scan';
 import partition from 'lodash/partition';
 import { default as BN } from 'bignumber.js';
-import { bigNumberify } from 'ethers/utils';
+import { bigNumberify, BigNumber } from 'ethers/utils';
```

Comment:
`bigify` uses `bignumber.js` and ethers version is a modified one. 
Let's create a type alias that we control. Like `TBN` ?
That way we have better control on the TS.

Reply:
We ended up not needing `bigify` in this because ethers.js has the `.isZero()` method already.
> Let's create a type alias that we control. Like TBN ?
That way we have better control on the TS.

Should this be it's own task, since it'll likely end up needing to be distributed throughout the `services/Store/` & `utils/` folders?